src/en/space-station-14/combat.md

```diff
@@ -1,5 +1,70 @@
-```admonish warning "Attention: Placeholder!"
-This section is a placeholder, pending a design-doc being created by the related work-group
-```
+# Combat Design Document
 
-# Combat
+## Concept
+
+Combat in SS14 is a core part of the experience, acting as a way to exert influence in the game world on the same level of importance as speech and interactions. It is by far the most brute-force method of doing so as the main goal of combat is to forcibly deprive another party of resources, whether that be health, time, opportunity or equipment. Because of this it is a natural enabler (and sometimes solution) for conflict, creating an underlying system for consequences that reverberates throughout the game.
+
+Combat should be possible at a range from low to high lethality, with the latter risking consequences for participants that may last for the remaining duration of the round. You should not have to be good at combat to play SS14, but combat will be something every player has to engage with to some degree on the station, whether that be through being an active combatant, desperate self-defense or just fleeing the danger.
+
+## Intended Experience
+
+- Intent matches consequences; a friendly boxing match is easily recovered from, a deadly gunfight is not.
+- Weapons matches lethality; a single punch can be shrugged off, a baseball bat hurts, several knife wounds kill.
+- Equipment matches role; the tools, tactics and equipment available to a role provides distinct gameplay differences that reinforce the role's strengths.
+- Viable and variable choices; the game provides meaningful choices in the equipment and strategies you choose to bring to combat. 
+- Reactable combat: you may not have time to do everything you want, but you should generally have time to respond with *something*. 
+- Centered on strategy; understanding how to strategize on both macro and micro levels of combat is rewarded, with adapting to others' strategies on the fly being key to success.
```

Comment:
Do you mean "tactics," as you discuss later in the docs? "Tactics" is how to win a fight when you're already in one. "Strategy" is who and when to fight, if at all.